model/src/main/java/uk/gov/hmcts/payment/api/service/LoggingCreditAccountPaymentService.java

```diff
@@ -77,4 +77,10 @@ public List<PaymentFeeLink> search(Date startDate, Date endDate) {
         LOG.info("PaymentFeeLinks found: {}", paymentFeeLinks.size());
         return paymentFeeLinks;
     }
+
+    @Override
+    public void deleteByPaymentReference(String paymentReference) {
+        LOG.info("Deleting payment for reference: {}", paymentReference);
```

Comment:
Are there only records to be deleted from Payment, none from the other tables?

Reply:
All the records from all relational tables are deleted in cascade method

Reply:
resolved